notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Please a WARNINGS file in this directory and add an entry mentioning that the exact GPU setup this labs need to run on.</p><p>Remove the GPU test</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I will add a file after this. So far removed the file.
And can I ask you why `WARNING` file? Is it ok to put in `README.md`? @BenoitDherin